src/main/resources/avro/references.avdl

```diff
@@ -125,13 +100,21 @@ record Reference {
 
   /** ID from http://www.ncbi.nlm.nih.gov/taxonomy (e.g. 9606->human). */
   union { null, int } ncbiTaxonId = null;
+
+  /** whether this reference is primary, part of the core coordinate system, or not */
+  boolean isPrimary = true;
 }
 
 /**
 A `ReferenceSet` is a set of `Reference`s which typically comprise a
 reference assembly, such as `GRCh38`. A `ReferenceSet` defines a common
 coordinate space for comparing reference-aligned experimental data.
+
+In classic mode, this is just a set of sequences.  In graph mode, there
+are both sequences and joins, but we only access the joins via method
+getReferenceJoins().
 */
+
 record ReferenceSet {
   /** The reference set ID. Unique in the repository. */
```

Comment:
@adamnovak oops... was accidentally searching on master, not your PR version of the file. Sorry!